server/conf/docker/Dockerfile

```diff
@@ -7,7 +7,7 @@ ARG DEBIAN_FRONTEND=noninteractive
 EXPOSE 8000
 
 # https://python-poetry.org/docs/configuration/#using-environment-variables
-ENV POETRY_VERSION=1.2.0 \
+ENV POETRY_VERSION=1.3.2 \
```

Comment:
Yeah, I had to reinstall Poetry on my new laptop and v1.3 produces lockfiles that are incompatible with v1.2. I'll undo the change in this diff though since I ended up reverting the requirement changes I thought I needed.